### ISSUE TITLE
UI Mockup

### ISSUE THREAD

**Issue by mastercoms** [CircuitSoftGroup/WaveCloud#2]
We need to design a hand sketched mockup for the UI.

Beforehand, we must also discuss the elements and pages we must design.

**Comment by steampunkgiraffe** [comment]
The home page where all the files should be the first thing we prioritize on in terms of ui design

**Comment by mastercoms** [comment]
Yeah, and I think then we should design how viewing files would look like for a generic file, and then we can work on specific file types like documents, videos, and pictures.

**Comment by mastercoms** [comment]
The pages/components I think are necessary:

* **Landing** - features, call to action, pricing
* **Home view** - pinned items, items listed by relevance (frecency), with a toggle to switch to a details view, which would be sorted by name by default, and is switchable by the user
* Upload UI - create a new file, upload a file, along with options to set an expiration trigger (number of downloads, date past, etc)
 * Download UI - download 1 file, download multiple files
* Share UI

Components that do not have priority:

* Preview UI - preview any file type

Anything else you think I may have missed?

**Comment by mastercoms** [comment]
I'm delaying the UI to July 18 because we can use ownCloud's UI for now and it will allow us to better design our own UI.
